### ISSUE TITLE
Remove references to accessibility issues in Frontend Docs from accessibility statement [before 31 March]

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-design-system#1569]
## What

Once https://github.com/alphagov/govuk-frontend-docs/issues/104 has been merged and deployed, update the accessibility statement to remove the reference to issues caused by the technical documentation template, and update the compliance status.

## Why

The known accessibility issues should be resolved by updating to v2.2.0 of the template and ensuring tables have the correct row headers (https://github.com/alphagov/govuk-frontend-docs/issues/104)

## Done when

- [x] References to issues in Frontend Docs removed
- [x] Compliance status updated
- [ ] Changes 2i'ed

**Comment by EoinShaughnessy** [comment]
As discussed in the tech writers' docs catchup, the actions to update accessibility statements are as follows:

- In your product team's tech doc accessibility statement:
  - remove the reference to the tech doc template from the "How accessible this website is" section
  - remove the reference to the tech doc template from the "Non-compliance with the accessibility regulations"
  - in the "How we tested this website", update the last tested date and remove the bulleted list
  - update the "What we’re doing to improve accessibility" section
  - in the "Preparation of this accessibility statement", update the last reviewed and last tested dates
 
If the documentation is now fully compliant with the regulations, reflect this in the statement using the phrase "This website is fully compliant with the Web Content Accessibility Guidelines version 2.1 AA standard" in the following sections:
- How accessible this website is
- Compliance status
- What we're doing to improve accessibility - suggest you put something in about maintaining the fully compliant status

Check the [sample accessibility statement](https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website) here for more information if needed.

**Comment by 36degrees** [comment]
We'll need to adjust that slightly as the accessibility statement for the frontend docs is combined with the accessibility statement for the Design System.

**Comment by EoinShaughnessy** [comment]
@36degrees Just to confirm:

Is this issue to update https://design-system.service.gov.uk/accessibility/?

And is https://github.com/alphagov/prototype-kit-training/issues/53, because it's in the prototype-kit-training repo, to updatehttps://prototype-kit-training.cloudapps.digital/accessibility.html#non-compliance-with-the-accessibility-regulations?
